.travis.yml

```diff
@@ -13,6 +13,11 @@ install:
   - composer install
 
 script:
+  # Static analyzer check
+  - if [ $TRAVIS_PHP_VERSION = '7.1' ]; then composer require "phpstan/phpstan:^0.9" && ./vendor/bin/phpstan analyze -c .phpstan.neon --level=2 --no-progress src;fi
+  # Check the code style
+  - if [ $TRAVIS_PHP_VERSION = '7.1' ]; then IFS=$'\n'; COMMIT_SCA_FILES=($(git diff --name-only --diff-filter=ACMRTUXB "${TRAVIS_COMMIT_RANGE}")); unset IFS; fi
```

Comment:
Why is this needed?

Reply:
This added to run static analyze checks and code style checks, so code with wrong style-guide, some misspelling, incorrect return types and phpdocs will not be merged to master branch. And it should be run once, not for each php version. The instruction for php-cs-fixer was taken from [documentation](https://github.com/FriendsOfPHP/PHP-CS-Fixer#using-php-cs-fixer-on-ci).